docs/common-toolchains.md

```diff
@@ -154,11 +155,12 @@ most recent revision of the common toolchains at that time.
 |---------|----------|------------|--------|-------------------|-------------|-------------|
 | `2019a` | Jan '19  | 2.31.1     | 8.2.0  | 2019.1.144        | 2018.4.274  | 2019.1.144  |
 | `2019b` | Sept '19 | 2.32       | 8.3.0  | 2019.5.281        | 2018.5.288  | 2019.5.281  |
-| `2020a` | May '20   | 2.34       | 9.3.0  | 2020.1.217        | 2019.7.217  | 2020.1.217  |
-| `2020b` | Nov '20   | 2.35       | 10.2.0 | 2020.4.304        | 2019.9.304  | 2020.4.304  |
-| `2021a` | May '21   | 2.36.1     | 10.3.0 | 2021.2.0          | 2021.2.0    | 2021.2.0    |
-| `2021b` | Oct '21   | 2.37       | 11.2.0 | 2021.4.0          | 2021.4.0    | 2021.4.0    |
-| `2022a` | Jun '22   | 2.38       | 11.3.0 | 2022.1.0          | 2021.6.0    | 2022.1.0    |
+| `2020a` | May '20   | 2.34      | 9.3.0  | 2020.1.217        | 2019.7.217  | 2020.1.217  |
+| `2020b` | Nov '20   | 2.35      | 10.2.0 | 2020.4.304        | 2019.9.304  | 2020.4.304  |
+| `2021a` | May '21   | 2.36.1    | 10.3.0 | 2021.2.0          | 2021.2.0    | 2021.2.0    |
+| `2021b` | Oct '21   | 2.37      | 11.2.0 | 2021.4.0          | 2021.4.0    | 2021.4.0    |
+| `2022a` | Jun '22   | 2.38      | 11.3.0 | 2022.1.0          | 2021.6.0    | 2022.1.0    |
+| `2022b` | Dec'22    | 2.39      | 12.2.0 | 2022.2.1          | 2021.7.1    | 2022.2.1    |
```

Comment:
```suggestion
| `2020a` | May '20   | 2.34       | 9.3.0  | 2020.1.217        | 2019.7.217  | 2020.1.217  |
| `2020b` | Nov '20   | 2.35       | 10.2.0 | 2020.4.304        | 2019.9.304  | 2020.4.304  |
| `2021a` | May '21   | 2.36.1     | 10.3.0 | 2021.2.0          | 2021.2.0    | 2021.2.0    |
| `2021b` | Oct '21   | 2.37       | 11.2.0 | 2021.4.0          | 2021.4.0    | 2021.4.0    |
| `2022a` | Jun '22   | 2.38       | 11.3.0 | 2022.1.0          | 2021.6.0    | 2022.1.0    |
| `2022b` | Dec '22   | 2.39       | 12.2.0 | 2022.2.1          | 2021.7.1    | 2022.2.1    |
```

Make fewer changes compared to the existing.

During the conversion we added a space in the dates in this table. So `Dec '22` instead of `Dec'22`. This makes it consistent with the dates in the `foss` table.